README.md

```diff
@@ -556,7 +555,7 @@ status:
 
 1.  Mount an entire `ConfigMap` as the binding `Secret`
 
-	```yaml
+	  ```yaml
```

Comment:
This might create rendering problem here: https://k8s-service-bindings.github.io/spec/

Reply:
the indentation was causing issues with the TOC generation. Looks like the root cause is the tab.